rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -951,7 +951,7 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> guiSettings, std:
 
 	// Sliders
 
-	EnhanceSlider(emu_settings::DriverWakeUpDelay, ui->wakeupDelay, ui->wakeupText, tr(u8"%0 µs"));
+	EnhanceSlider(emu_settings::DriverWakeUpDelay, ui->wakeupDelay, ui->wakeupText, tr(reinterpret_cast<const char*>(u8"%0 µs")));
```

Comment:
why is the u8 even necessary

Reply:
Without u8, it's a "string in whatever encoding compiler finds necessary". Even if file is UTF-8.

Reply:
so it's because of the µ ?

Reply:
also, I think you could also use QChar() if i'm correct

Reply:
Yes, idk about QChar

Reply:
Can just use some overloaded operator""(x) for it.